mobile/src/components/course/course.tsx

```diff
@@ -58,6 +67,9 @@ const Course: FC = () => {
 
   useEffect(() => {
     if (course) {
+      dispatch(courseModulesActions.clearModules());
+      dispatch(coursesActions.clearTasks());
```

Comment:
Maybe we should do this in useFocusEffect in return?

Reply:
When I pass clear actions to useFocusEffect, after going back from module screen - it runs, so decided to move this logic when we go back from course screen to dashboard